src/app/app.routes.ts

```diff
@@ -1,9 +1,18 @@
 import { Routes } from '@angular/router';
+import { AppRouteSegment } from './app-route-segment';
 
 export const routes: Routes = [
   {
     path: '',
     loadComponent: () =>
       import('./pages/landing.page/landing.page.component').then((m) => m.LandingPageComponent),
   },
+
+  {
+    path: AppRouteSegment.PRODUCT + '/:id',
```

Comment:
NIT: I'd rather use a string literal as that is less prone to errors.

```js
`${AppRouteSegment.PRODUCT}/:id`
```

Reply:
Also, it should show the product-list on "/products" or "" in our case and a specific product on ":id" so it is not necessary to do a string concatenation